### PR TITLE
Enable VB Symbol tests on Linux CoreClr

### PR DESCRIPTION
**Pull request by jaredpar**
Part of #29399

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-compiler for review.

File: build/scripts/tests.sh [hunk 1]
```diff
@@ -82,7 +81,7 @@ do
     fi
 
     # https://github.com/dotnet/roslyn/issues/29380
-    if ${runner} "${xunit_console}" "${file_name[@]}" -xml "${log_file}" -parallel none -maxThreads 2
```
Comment (jaredpar) [inline build/scripts/tests.sh]: The use of `-maxThreads 2` was a testing mistake that made it into the last PR. Given we aren't running in parallel this really has no effect.

File: src/Compilers/VisualBasic/Test/Symbol/DocumentationComments/DocCommentTests.vb [hunk 2]
```diff
@@ -12197,7 +12197,9 @@ xmlDoc)
                         xmlDoc = CStr(stringMapper(xmlDoc))
                     End If
 
-                    Assert.Equal(expectedDocXml.Value.Replace(vbLf, vbCrLf).Trim(), xmlDoc)
+                    Assert.Equal(
+                        expectedDocXml.Value.Replace(vbLf, vbCrLf).Trim(),
+                        xmlDoc.Replace(vbLf, vbCrLf).Trim())
```
Comment (chsienki) [inline src/Compilers/VisualBasic/Test/Symbol/DocumentationComments/DocCommentTests.vb]: This breaks the windows tests. It replaces \r\n with \r\r\n 

You could replace vbCrLf -> vbLf -> vbCrLf to make it work for both?

Comment (jaredpar) [reply src/Compilers/VisualBasic/Test/Symbol/DocumentationComments/DocCommentTests.vb]: Ah yes. Will fix that.

Comment (jaredpar) [reply src/Compilers/VisualBasic/Test/Symbol/DocumentationComments/DocCommentTests.vb]: Actually even simpler idea here. The `expectedDocXml` is guaranteed to have `vbLf` newlines as VB XML literals strictly follow the XML specification. I can just translate `xmlDoc.Replace(vbCrlf, vbLf)` and that should be sufficient.

**Comment by jaredpar** [comment]
retest this please

**Comment by jaredpar** [comment]
Jenkins appeared hung. Requeued the tests.

**Comment by jaredpar** [comment]
@chsienki, @333fred issues should be resolved now.

**Comment by MattGal** [comment]
@dotnet-bot test windows_release_vs-integration_prtest  please

**Pull request merged**